### DISCUSSION TITLE
afl-cmin reduced coverage

### DISCUSSION THREAD

**Discussion by p1-leo** [AFLplusplus/AFLplusplus#1872] [Q&A] [answered]
Hello,

I have a question regarding the corpus trimming with afl-cmin, from my understanding afl-cmin should keep any file that take a different path in the binary and thus should represent the certain code coverage.
The issue that I'm facing is that after using afl-cmin, if I replay the remaining corpus file, the coverage is reduced in comparison with the full corpus.
My bet is that I have an issue somewhere in the compilation of my fuzzer,
but I would like to know if it is possible for afl-cmin in a standard usage to remove some of the coverage.

Thanks you, and thanks to the maintainer of this amazing tool.

**Comment by vanhauser-thc** [comment, marked as answer]
Are you using a current version?
There was a bug in afl-cmin where a bit of coverage was lost, I fixed that 6+ months ago I think

**Reply by p1-leo** [reply]
Thank you very much, I checked the version, and it was the last one, but I got my answer `afl-cmin should not lose coverage`.
I'm a little embarrassed, it was simple bug in a script that is used to gather all corpus from parallel fuzzer, and some file were missing.
